vehicle/mkvi/software/bms/utils/fault.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+#include "vehicle/mkvi/software/bms/can_api.h"
+
+// enum FaultCode {
```

Comment:
You can delete all this stuff because it's now taken care of by the CAN API! See #170.

Reply:
I'd say you don't need the enum in here. Maybe leave a comment indicating that the list of faults is defined in the `bms.yml` file.